static/js/place/place.ts

```diff
@@ -129,6 +129,12 @@ async function getLandingPageData(dcid: string): Promise<PageData> {
   });
 }
 
+function shouldMakeChoroplethCalls(dcid: string, placeType: string): boolean {
+  const isInUSA: boolean =
+    dcid.startsWith("geoId") || dcid.startsWith("country/USA");
+  return isInUSA && (placeType == "Country" || placeType == "State");
```

Comment:
we should also continue adding it for country (sorry if that unclear)

Reply:
Sorry, what do you mean? This check makes sure we are calling choropleth apis for countries and states that are in the USA